package.json

```diff
@@ -5,7 +5,7 @@
   "version": "3.9.0",
   "author": "Hiro Systems PBC",
   "scripts": {
-    "dev": "concurrently --raw 'node webpack/dev-server.js' 'redux-devtools --hostname=localhost --port=8000'",
+    "dev": "concurrently --raw 'node -r dotenv/config webpack/dev-server.js' 'redux-devtools --hostname=localhost --port=8000'",
```

Comment:
do we still need the dotenv here?

Reply:
I can't ever get my local .env file to load the env variables if I don't add it. They are undefined unless I add this.